<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>350</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>75</weight>
    <italic>false</italic>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>-320</y>
     <width>271</width>
     <height>701</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>25</pointsize>
     <weight>75</weight>
     <italic>true</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Alta pacients</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>270</y>
     <width>294</width>
     <height>71</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="hlButtons">
    <item>
     <widget class="QPushButton" name="pbInserir">
      <property name="styleSheet">
       <string notr="true">font: italic 13pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Inserir pacient</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pbModificar">
      <property name="styleSheet">
       <string notr="true">font: italic 13pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Modificar pacient</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>71</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="lDni">
      <property name="styleSheet">
       <string notr="true">font: italic 13pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Dni</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lNom">
      <property name="styleSheet">
       <string notr="true">font: italic 13pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lCognom">
      <property name="styleSheet">
       <string notr="true">font: italic 13pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Cognom</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lAltura">
      <property name="styleSheet">
       <string notr="true">font: italic 13pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Altura</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="lPes">
      <property name="styleSheet">
       <string notr="true">font: italic 13pt &quot;Arial&quot;;</string>
      </property>
      <property name="text">
       <string>Pes</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLineEdit" name="textNom">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>151</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textCognom">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>150</y>
     <width>151</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textDni">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>70</y>
     <width>151</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textPes">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>230</y>
     <width>151</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textAltura">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>190</y>
     <width>151</width>
     <height>34</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
